ner_v2/detectors/temporal/date/date_detection.py

```diff
@@ -8,27 +8,17 @@
 import pytz
 
 import models.crf.constant as model_constant
-import ner_v2.constant as detector_constant
+import ner_v2.detectors.temporal.constant
```

Comment:
Minor:
Import as `import ner_v2.detectors.temporal.constant as temporal_constant`. Makes the code shorter

Reply:
Make sense, done